setup.py

```diff
@@ -0,0 +1,23 @@
+from setuptools import setup, find_packages
+from Cython.Build import cythonize
+
+setup(
+    name='Python Examples',
+    version='0.0.1',
+    description='Python example code',
+    url='https://github.com/w0251251/python_examples',
+    author='Nicholas Tancredi',
+    author_email='nicholastancredi@gmail.com',
+    license='GPL3',
+    packages=find_packages(),
+    ext_modules=cythonize("examples/cython_array_sum/index.pyx"),
```

Comment:
any documentation around what this does?

Reply:
https://cython.readthedocs.io/en/latest/src/tutorial/cython_tutorial.html